app/test_main.py

```diff
@@ -1,3 +1,21 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_if_age_is_zero():
+    assert get_human_age(0, 0) == [0, 0]
+
+
+def test_if_age_is_less_than_15():
+    assert get_human_age(14, 14) == [0, 0]
+
+
+def test_if_age_is_more_than_15():
+    assert get_human_age(23, 23) == [1, 1]
+
+
+def test_if_age_is_more_than_24():
+    assert get_human_age(27, 27) == [2, 2]
+
+
+def test_if_age_is_very_old():
```

Comment:
"very old" is  an ambiguous name. Recommend changing to something more specific.